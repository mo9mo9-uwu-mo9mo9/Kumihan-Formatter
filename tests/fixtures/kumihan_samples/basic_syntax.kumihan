# 見出し #基本的な見出しサンプル##

これは通常のテキストです。

# 太字 #重要な情報##
# イタリック #斜体のテキスト##

段落の区切りもテストします。

# リスト #
- 項目1
- 項目2
- 項目3
##

# ネスト #複雑な# 太字 #ネスト##した記法##