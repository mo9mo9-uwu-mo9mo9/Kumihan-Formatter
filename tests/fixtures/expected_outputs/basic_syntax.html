<h1>基本的な見出しサンプル</h1>

<p>これは通常のテキストです。</p>

<p><strong>重要な情報</strong></p>
<p><em>斜体のテキスト</em></p>

<p>段落の区切りもテストします。</p>

<ul>
<li>項目1</li>
<li>項目2</li>
<li>項目3</li>
</ul>

<p>複雑な<strong>ネスト</strong>した記法</p>